spec/wor/test_services/some_service_failure_spec.rb

```diff
@@ -110,6 +110,18 @@
           expect { service.method_without_rescue }.to raise_error(Wor::Requests::RequestError)
         end
       end
+
+      context 'when base_url malformed defined' do
+        subject(:service) { described_class.new(base_url_empty: true) }
+
+        before do
+          Spy.spy(service.logger, 'info')
+        end
+
+        it 'returns error with developer message' do
+          expect { service.method_with_log_with_block }.to raise_error
```

Comment:
Just a comment, some times the linter complains about the explicity of the error, in this case you should declared that is a `NoMethodError`.
But is OK!

Reply:
In this rubocop version there is no problem, but in the future I want to upgrade rubocop to improve this and others